src/Hero6.UserInterface.SierraVga/Entities/Padding.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
```

Comment:
Alternatively, none of these using directives are actually used, so you can just remove them.